sale_order_line_clone/security/sale_order_line_clone_security.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+        <record model="res.groups" id="group_can_clone_sale_line">
+            <field name="name">Can Clone Sale Order Line</field>
+            <field name="category_id" ref="base.module_category_usability" />
+            <field
+            name="users"
```

Comment:
This should have one more tabulation

Reply:
Changed

Reply:
Sorry, the tabulation was between lines 6 and 9 (included), not between 3 and 10

Reply:
@RogerSans Can you review this?